src/helpers/get-changed-files.ts

```diff
@@ -11,10 +11,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import { HelperInputs } from '../types/generated';
 import { context } from '@actions/github';
 import { getChangedFilepaths } from '../utils/get-changed-filepaths';
 
-export const getChangedFiles = async () => {
+export class GetChangedFiles extends HelperInputs {
+  glob_filter_paths?: string;
+}
+
+export const getChangedFiles = async ({ glob_filter_paths }: GetChangedFiles) => {
   const filePaths = await getChangedFilepaths(context.issue.number);
-  return filePaths.join(',');
+  const filteredFilePaths = glob_filter_paths
+    ? filePaths.map(fileName => fileName.match(glob_filter_paths))?.filter(localizationFile => localizationFile !== null)
```

Comment:
Just realized you are looking for a regex pattern, not a glob pattern, right? There is an existing input called `pattern` for this
```suggestion
    ? filePaths.filter(fileName => fileName.test(pattern))
```